packages/rizin/build.sh

```diff
@@ -0,0 +1,9 @@
+# Thank you @TomJo2000
```

Comment:
Please remove this comment. You can put TomJo as co-developer instead in commit message, like:

```
rizin: add package

Co-developed-by: Joshua Kahn @TomJo2000
```